cmd/kni-install/create.go

```diff
@@ -108,8 +109,17 @@ var (
 					logrus.Fatal(err)
 				}
 
-				logrus.Warn("FIXME! Exiting after bootstrap, remove when all operators will come up successfully.")
-				return
+				// FIXME: baremetal platform needs to perform manual steps after bootstrap completes, so the install
+				// exits early. See https://github.com/openshift-metal3/kni-installer/issues/60
+				if assetStore, err := assetstore.NewStore(rootOpts.dir); err == nil {
+					platformConfig := &installconfig.InstallConfig{}
+					if err := assetStore.Fetch(platformConfig); err == nil {
+						if platformConfig.Config.BareMetal != nil {
```

Comment:
Looking at the code I'm not sure this will work, we need to test platformConfig.Config.Platform.Name() I think?

See:

https://github.com/openshift-metal3/kni-installer/blob/master/pkg/asset/installconfig/installconfig.go#L69..L75

https://github.com/openshift-metal3/kni-installer/blob/master/pkg/asset/cluster/metadata.go#L63